src/client/components/InputBar/InputBar.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { addItem } from './actions';
+import MessageBox from '../MessageBox';
+
+class SearchBar extends Component {
+  state = {
+    itemAuthor: '',
+    itemName: ''
+  };
+
+  handleNameChange = e => {
+    this.setState({
+      itemName: e.target.value
+    });
+  };
+
+  handleAuthorChange = e => {
+    this.setState({
+      itemAuthor: e.target.value
+    });
+  };
+
+  handleFormSubmit = e => {
+    e.preventDefault();
+    const { addItem } = this.props;
+    const { itemAuthor, itemName } = this.state;
+    const newItem = {
+      author: itemAuthor,
+      isOrdered: false,
+      name: itemName
+    };
+
+    addItem(newItem)
+      .then(() => {
+        this.setState({
+          itemAuthor: '',
+          itemName: ''
+        });
+      })
+      .catch(err => console.error(err.message));
```

Comment:
shouldn't this error be shown in UI?

Reply:
Error shown in UI is handled in actions by dispatching ` dispatch(addItemError(err));`

Here inside the component, I need this catch statement to not reset state if an error occurs, 
if I remove it all inputs are being reset anyway. 
Maybe I sholudn't console.log this error? - but if so, what to do inside this catch statement?